lib/vagrant-digitalocean/actions/create.rb

```diff
@@ -40,6 +40,14 @@ def call(env)
           # assign the machine id for reference in other commands
           @machine.id = result['droplet']['id'].to_s
 
+          # assign a specific floating IP to this droplet
+          if @machine.provider_config.floating_ip
+            result = @client.post("/v2/floating_ips/#{@machine.provider_config.floating_ip}/actions", {
+              :type => 'assign',
+              :droplet_id => @machine.id,
```

Comment:
There is a hanging `,` here that likely is preventing this from running?

Reply:
I don't code in Ruby (I quickly knocked this up to serve a purpose), but my understanding is that a trailing comma is purely stylistic and doesn't affect the functionality of the code.

I have added a trailing comma out of habit (to reduce line noise in MRs) but I'm using this same code on my box to provision digital ocean containers without a problem.

I think this repo might be stale. If I had feedback to tell me to drop the comma I will, but until then it will continue to knock at the PR door.

Reply:
We're still here! We've been working on partnering with DigitalOcean to get this plugin up-to-date with their product suite. Until then, there will be some delay on features like this.

Reply:
Ack. I couldn't get this fix to work for me, so likely needs some closer scrutiny.
What did work was a curl script in the provisioning stage - but without  DO's input it'll be painful working out a robust solution.